### PR TITLE
Protocol Buffer update for release 1.13.x at Thu Jan 28 08:49:34 UTC 2021

### PR DESCRIPTION
**Pull request by Corona-Warn-App-Technical-User**

### CODE REVIEW

**Comment by mlenkeit** [comment]
Closing; not relevant for this repo.

File: common/protocols/src/main/proto/app/coronawarn/server/common/protocols/internal/v2/app_config_android.proto [hunk 1]
```diff
@@ -25,6 +26,8 @@ message ApplicationConfigurationAndroid {
 
   DiagnosisKeysDataMapping diagnosisKeysDataMapping = 8;
   DailySummariesConfig dailySummariesConfig = 9;
+
+  PPDDEventDrivenUserSurveyParametersAndroid eventDrivenUserSurveyParameters = 10;
```
Comment (hilmarf) [inline common/protocols/src/main/proto/app/coronawarn/server/common/protocols/internal/v2/app_config_android.proto:30]: @mlenkeit but the app-config changes are relevant for this repo, aren't they!?

Comment (mlenkeit) [reply common/protocols/src/main/proto/app/coronawarn/server/common/protocols/internal/v2/app_config_android.proto:30]: good point!

**Comment by mlenkeit** [comment]
replaced by #1167

**Pull request closed without merging**